code/src/index.css

```diff
@@ -1,13 +1,169 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  margin: 20px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+header {
+  margin: 0 auto;
+}
+
+.header {
+  margin-bottom: 0px;
+  align-self: center;
+  border-bottom: 1px solid black;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 350px;
+  min-height: 550px;
+  border-radius: 2%;
+  border: 1px solid black;
+  background-color: rgba(242, 181, 90, 0.4);
+
+  display: flex;
+  flex-direction: column;
+  position: relative;
```

Comment:
This part is so well written. It's clean and does the job if you know what I mean. Like 101 how to center and make all the elements appear where you want!